bridges/TwitterV2Bridge.php

```diff
@@ -470,6 +470,24 @@ public function collectData()
 
             $this->item['title'] = $titleText;
 
+            // Get external link info
+            $extURL = null;
+            if (isset($tweet->entities->urls) && strpos($tweet->entities->urls[0]->expanded_url, 'twitter.com') === false) {
```

Comment:
A tip is to use `str_starts_with` here instead of strpos. strpos has some false negatives here if twitter.com is in the url.

Reply:
False negatives, as it it will return false even if twitter.com is in the expanded_url string?

Not sure that `str_starts_with` would work. twitter.com isn't at the start of the string, it comes after the https://. And it would be pic.twitter.com or some other twitter subdomain. Maybe `preg_match()` instead?

Reply:
I think we want this:
```
if (str_starts_with($tweet->entities->urls[0]->expanded_url, 'https://twitter.com/'))
```